labgrid/driver/commandmixin.py

```diff
@@ -13,6 +13,7 @@ class CommandMixin:
 
     def __attrs_post_init__(self):
         super().__attrs_post_init__()
+        self.decodeerrors = 'strict'
```

Comment:
This is not an attrs attribute, so it can't be specified during construction. What is the reason for the difference?